doc/Analytics.xml

```diff
@@ -3397,127 +3397,71 @@ Added Recognition rule engines</revremark>
     </section>
     <section>
       <title>Counting rule</title>
-      <para>Counting rule counts the number of motion object passing through the set of line segments (barrier) and optional direction attribute. The configuration parameters also include the time interval to report the events and time interval to reset its counter. </para>
+      <para>Counting rule counts the number of objects passing through a set of line segments
+        (barriers) with an optional direction. The configuration parameters also include the time
+        interval to report the events and time interval to reset its counter.<variablelist>
+          <varlistentry>
+            <term>Parameters</term>
+            <listitem>
+              <para role="param">Segments [tt:Polyline]</para>
+              <para role="text">One or more polylines used for detection. An object must have passed
+                through all of them to register as a count. ??? or ??? Passing through any of the
+                lines registers as a count. ???</para>
+              <para>ReportTimeInterval [xs:duration]</para>
+              <para>Optional time interval to report count information, starting from midnight. For
+                example, if the interval is set to PT2H, reports will be published at 2:00 am, 4:00
+                am, etc. Default is same as ResetTimeInterval.</para>
+              <para>ResetTimeInterval [xs:duration]</para>
+              <para>Periodic count reset time starting from midnight. For example, if the interval
+                is set to PT12H, the count will be reset at 12:00 noon and 12:00 midnight.</para>
+              <para>Direction [tt:Direction]</para>
```

Comment:
Our expectation for direction any is that both directions count up. Maybe we need to add two Any-modes

Reply:
Then the occupancy use-case is not possible. On the last call I thought that was what the majority wanted. Then I propose to use two counters again.

Reply:
Agree with Hans opinion, when the direction is any count should increment both ways.